packages/machine/test/integration/test-io-provider.ts

```diff
@@ -112,19 +105,17 @@ export class TestIOProvider {
     next: Function,
     context: Context
   ) {
-    const msg = getLastResult(Opcode.IO_PREPARE_SEND, context.results);
-
-    // FIXME: (ts-strict) msg should never be null here
-    // https://github.com/counterfactual/monorepo/issues/94
-    const value = msg.value;
-
-    // Hack for testing and demo purposes, full IO handling by client goes here
+    const value = context.intermediateResults.outbox!
```

Comment:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll Do we need to control this manually? Are there circumstances where `value` can be undefined? Maybe we can prevent it with better typing?